LICENSE.md

```diff
@@ -1,11 +1,54 @@
-# Licence
+# License
+
+## Analysis report
+
+Copyright (c) 2024
+
+This report (source contained in the [`diabetes_predictor_py` repository](https://github.com/UBC-MDS/diabetes_predictor_py/tree/main/analysis)) is made available under the **Attribution-NonCommercial-ShareAlike 4.0 International** ([CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)). 
+
+This is a human-readable summary of (and not a substitute for) the [license](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode).
+
+## You are free to:
+
+- **Share** — copy and redistribute the material in any medium or format
+- **Adapt** — remix, transform, and build upon the material
+
+The licensor cannot revoke these freedoms as long as you follow the license terms.
+
+## Under the following terms:
+
+- **Attribution** — You must give appropriate credit (mentioning that your work is derived from work that is Copyright © Inder Khera, Jenny Zhang, Jessica Kuo, and Javier Martinez), provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+- **NonCommercial** — You may not use the material for commercial purposes.
+- **ShareAlike** — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.
+
+**No additional restrictions** — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits.
+
+## Notices:
+
+You do not have to comply with the license for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation.
+
+No warranties are given. The license may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material.
+
+## Software code
 
 MIT License
 
-Copyright (c) 2024 Master of Data Science at the University of British Columbia
+Copyright (c) 2024 Inder Khera & Jenny Zhang
```

Comment:
I think this applies to the whole report? if so we should probably add everyone's name on the list.

Reply:
the MIT license specifically refers to the code, the other license on top is for the report and has everyone's names.

Same with Tiff's example. She include the whole team for the other license and onlhy her name for the coding part.

Reply:
gotcha! then it's all good.